src/state/slices/portfolioSlice/utils.test.ts

```diff
@@ -127,3 +317,26 @@ describe('findAccountsByAssetId', () => {
     expect(result2).toEqual([btcAccountId])
   })
 })
+
+describe('accountToPortfolio', () => {
```

Comment:
Probably want to use snapshot tests for these tests as stated above.

Reply:
Done